src/Visualizer/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
it would be better to add ignore iterms in root .gitignore file.

Reply:
Thank you for the feedback. 🙏